wiki/Help_Centre/en.md

```diff
@@ -34,13 +34,13 @@ If you are restricted, make sure to sort it out with support instead of just mak
 
 **Accounts are generally only disabled when they are confirmed to be multiple accounts belonging to the same user.**
 
-If this is the case for you, as in, the account that you have had banned was not the first one you ever made, then you can usually safely log in to your first registered account without problem. Generally speaking, disabled accounts are not reactivated a vast majority of the time.
+If this is the case for you, as in, the account that you have had banned was not the first one you ever made, then you can usually safely sign in to your first registered account without problem. Generally speaking, disabled accounts are not reactivated a vast majority of the time.
 
 If your account has been restricted, you'll likely know why after a brief conversation with the support team. 
 
 In situations where we are absolutely assured that foul play or wrongdoing has taken place, we will ask you to wait out a period of **three (3) months** from the date of your last offense before we consider an appeal from you to let you back into the game.
 
-### Login
+### Sign in
```

Comment:
probably revert

Reply:
I can add a dash (`Sign-in`) to make it a noun.

Reply:
:+1: